server/app/services/applicant/question/CurrencyQuestion.java

```diff
@@ -36,6 +36,19 @@ public ImmutableList<Path> getAllPaths() {
     return ImmutableList.of(getCurrencyPath());
   }
 
+  @Override
+  public ImmutableMap<Path, Double> getJsonEntries() {
+    Path path = getCurrencyPath().asApplicationPath().replacingLastSegment("currency_dollars");
+
+    if (getCurrencyValue().isPresent()) {
+      Long centsTotal = Long.valueOf(getCurrencyValue().get().getCents());
+
+      return ImmutableMap.of(path, centsTotal.doubleValue() / 100.0);
+    } else {
+      return ImmutableMap.of();
+    }
+  }
+
   @Override
```

Comment:
Above code is taken from `case CURRENCY`:

https://github.com/civiform/civiform/blob/358ad3dbb6e67aacb91afb3ede60e32ac281ba26/server/app/services/export/JsonExporter.java#L121-L219